walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -115,7 +141,8 @@
   private void addPossibleTime(int start, int end, boolean inclusive, long durationOfMeeting, Collection<TimeRange> possibleTimes) {
     TimeRange possibleTime = TimeRange.fromStartEnd(start, end, inclusive);
     if (possibleTime.duration() >= durationOfMeeting) {
-      possibleTimes.add(possibleTime);
+      if (!possibleTimes.contains(possibleTime))
```

Comment:
can we combine this into one if? 
`if (possibleTime.duration() >= duraitonOfMeeting && !possibletimes.contains(possibleTime))`